DeliveryApp.DataLayer/Models/Package.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DeliveryApp.DataLayer.Models
+{
+    public enum Size
+    {
+        SMALL = 15,
+        MEDIUM = 50,
+        LARGE = 150
+    }
+
+    public enum Status
+    {
+        PENDING_SENDING,
```

Comment:
Just, as the values are being stored in the database, i recommend assigning fixed values to each of the enum